runtime/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp

```diff
@@ -315,150 +305,72 @@ static void regclass_graph_OutputTensorInfo(py::module m) {
     out.def(py::init<size_t>(), R"(Constructor with parameter index as argument)");
     out.def(py::init<const std::string&>(), R"(Constructor with tensor name as argument)");
 
-    out.def(
-        "tensor",
-        [](const std::shared_ptr<ov::preprocess::OutputInfo>& me,
-           const std::shared_ptr<ov::preprocess::OutputTensorInfo>& outputTensorInfo) {
-            me->tensor(std::move(*outputTensorInfo));
-            return me;
-        },
-        py::arg("tensor"),
-        R"(
-                Adds builder for actual tensor information of client's output.
-                Parameters
-                ----------
-                tensor : OutputTensorInfo
-                    Client's output tensor information. It's internal data will be moved to parent OutputInfo object.
-                Returns
-                ----------
-                tensor : OutputInfo
-                    Reference to itself to allow chaining of calls in client's code in a builder-like manner.
-              )");
-    out.def(
-        "postprocess",
-        [](const std::shared_ptr<ov::preprocess::OutputInfo>& me,
-           const std::shared_ptr<ov::preprocess::PostProcessSteps>& postProcessSteps) {
-            me->postprocess(std::move(*postProcessSteps));
-            return me;
-        },
-        py::arg("post_process_steps"),
-        R"(
-                Adds builder for actual postprocessing steps for output parameter.
-                Parameters
-                ----------
-                post_process_steps : PostProcessSteps
-                    Postprocessing steps. It's internal data will be moved to parent OutputInfo object.
-                Returns
-                ----------
-                preprocess : OutputInfo
-                    Reference to itself to allow chaining of calls in client's code in a builder-like manner.
-              )");
-    out.def(
-        "network",
-        [](const std::shared_ptr<ov::preprocess::OutputInfo>& me,
-           const std::shared_ptr<ov::preprocess::OutputNetworkInfo>& outputNetworkInfo) {
-            me->network(std::move(*outputNetworkInfo));
-            return me;
-        },
-        py::arg("output_network_info"));
+    out.def("tensor", [](ov::preprocess::OutputInfo& me) {
+        return &me.tensor();
+    });
+    out.def("postprocess", [](ov::preprocess::OutputInfo& me) {
+        return &me.postprocess();
+    });
+    out.def("network", [](ov::preprocess::OutputInfo& me) {
+        return &me.network();
+    });
 }
 
 static void regclass_graph_OutputNetworkInfo(py::module m) {
-    py::class_<ov::preprocess::OutputNetworkInfo, std::shared_ptr<ov::preprocess::OutputNetworkInfo>> info(
+    py::class_<ov::preprocess::OutputNetworkInfo, ref_wrapper<ov::preprocess::OutputNetworkInfo>> info(
         m,
         "OutputNetworkInfo");
     info.doc() = "openvino.impl.preprocess.OutputNetworkInfo wraps ov::preprocess::OutputNetworkInfo";
 
     info.def(py::init<>());
```

Comment:
Please remove this 'init' as well - user doesn't need to create such objects (there is no place where it can be passed)

Reply:
Removed.